src/index.ts

```diff
@@ -3,3 +3,5 @@ export * from './auth/index.js';
 export * from './userinfo/index.js';
 export * from './lib/errors.js';
 export * from './lib/models.js';
+export * from './deprecations.js';
+export { Client, ClientCreate, ClientUpdate } from './deprecations.js';
```

Comment:
This overrides `export * from './lib/models.js';` because the explicitly named exports take priority